pkg/authorization/api/types.go

```diff
@@ -20,18 +20,53 @@ const (
 	VerbAll     = "*"
 )
 
+const (
+	ResourceGroupPrefix         = "group"
+	BuildGroupName              = ResourceGroupPrefix + ":builds"
+	DeploymentGroupName         = ResourceGroupPrefix + ":deployments"
+	ImageGroupName              = ResourceGroupPrefix + ":images"
+	OAUTHGroupName              = ResourceGroupPrefix + ":oauth"
```

Comment:
On a related note, since we're moving towards case-insensitivity for resource types, should we go ahead and do that in policy? I would be annoyed if my grant of `oauthAccessTokens` didn't take effect because of a lower-case `a`, especially when (some) other parts of the system let me refer to those objects that way